org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/wizards/PresetEditingWizardPreviewPage.java

```diff
@@ -102,6 +106,12 @@ private void createPreviewViewer(Composite container) {
 	}
 
 	private void populateUi() {
-		document.set("<jfragent>\n</jfragent>"); // TODO: generate real output preview from this.preset
+		document.set(preset.serialize());
```

Comment:
it would be good to make the xml preview scroll-able because it gets cuts off.

Reply:
fixed in bdd0a4f